firebase-devservices/deployment/src/main/java/io/quarkiverse/googlecloudservices/firebase/deployment/testcontainers/FirebaseEmulatorContainer.java

```diff
@@ -1101,58 +1101,66 @@ public ImageFromDockerfile build() {
         }
 
         private void buildBaseImageIfNeeded() {
+            LOGGER.info("Checking if we need to build a new base image for the Firebase dev service image");
             if (!baseImageExists()) {
                 var baseDockerFile = buildResource("Dockerfile.base");
 
+                var baseImageName = emulatorConfig.dockerConfig().imageName();
+                var user = dockerUser();
+                var group = dockerGroup();
+
+                LOGGER.debug("Using base image {}", baseImageName);
+                LOGGER.debug("Running as {} : {}", user, group);
+                LOGGER.debug("Installing firebase version: {}", firebaseVersion);
+
                 var baseImage = new ImageFromDockerfile(baseImageName(), false)
                         .withDockerfile(baseDockerFile)
                         .withFileFromPath("user-add.sh", buildResource("user-add.sh"))
-                        .withBuildArg("BASE_IMAGE", emulatorConfig.dockerConfig().imageName())
-                        .withBuildArg("USER_ID", "" + dockerUser())
-                        .withBuildArg("GROUP_ID", "" + dockerGroup())
+                        .withBuildArg("BASE_IMAGE", baseImageName)
+                        .withBuildArg("USER_ID", "" + user)
+                        .withBuildArg("GROUP_ID", "" + group)
                         .withBuildArg("FIREBASE_VERSION", firebaseVersion);
 
                 baseImage.get();
                 this.dockerBuilder = null;
+            } else {
+                LOGGER.info("Base image already exists. Using the cached version");
```

Comment:
```suggestion
                LOGGER.debug("Base image already exists. Using the cached version");
```

If you think it deserves a log, you should emit a single log with information in it, here there is 2 logs when the base image already exist and one otherwise.

Reply:
Fixed, added new log line when there is no image. Changed level. 

Squashed and force pushed